### ISSUE TITLE
Tinkers Construct Integration repair material

### ISSUE THREAD

**Issue by bblewitt** [BlakeBr0/MysticalAgradditions#72]
**Describe the bug**
When making a out of tinkers construct parts for Inferium and possible others that the ingot does not repair tools in the tickers anvil or tinkers station but the only way to repair tools is using repair kits.

**To Reproduce**
Mods installed is Tinkers construct, Mystical Agriculture and Mystical Aggradations
Have a working smeltery, get inferium ingots and make a pickaxe or any tool with the head made out of inferium and the handle made out of iron.
Use 1 or more bits of damage and place in the ingot to repair the tool and it shows no out put.

**Expected behavior**
Like other tinkers materials the expected behaviour is to have the ingot and/or the repair kit to repair the tool in the anvil or the tinker station.

**Screenshots / Scripts / Logs**
https://imgur.com/a/MVkMmd0 = Ingot
https://imgur.com/a/T2WeUh5 = Repair kit

**Versions (please complete the following information):**
 - Minecraft: 1.16.5
 - Forge: 36.2.2
 - Cucumber: 4.1.10
 - Mystical Agriculture: 4.2.1
 - Mystical Agradditions: 4.2.0

**Comment by AlgorithmLX** [comment]
This is not Bug, since you use a Tinker, then the Tinker's mechanics are used

**Comment by EdoTM** [comment]
Tinker's mechanics comprehend repairing a tool with the material of the head in the anvil or tinker station

**Comment by Lordmau5** [comment]
I've started playing FTB OceanBlock with a friend 2 days ago and we're running into the same issue.

We got a full Tertium Sword (Small Blade + 2 Tool Handles) and, from what I understand, we are supposed to repair it in a Tinker's Anvil with the Tertium Ingot.

This doesn't work though. The only way to repair it is, from what I can tell, to use a (in this case) Tertium Repair Kit.

I've tried with a full Iron pickaxe and I can repair it with an Iron Ingot just fine, so I feel this is an issue with the Tinker's implementation in Mystical Aggraditions / Agriculture.

I've also tried in a modpack with **just** Tinker's Construct, Mystical Agriculture and Mystical Aggraditions and even there (without any of those OceanBlock scripts) I can't repair the Tertium Sword with a Tertium Ingot.

![image](https://user-images.githubusercontent.com/1345036/135244150-7ff9519b-7e32-491c-8f4a-0d1ed3820da2.png)
![image](https://user-images.githubusercontent.com/1345036/135244175-b7c16c6d-9ab4-4bf2-b2ad-c9597adcb502.png)
